### ISSUE TITLE
FSLogix-Update/Install fails 

### ISSUE THREAD

**Issue by benni1972** [Deyda/NeverRed#73]
At least for Microsoft FSLogix 2.9.8784.63912 (Production)

FSLogix install fails due to file not found after downloading and extracting the downloaded ZIP.

It seem that in previous releases there was a Top-Level-Folder inside the ZIP archive named `"FSLogix_Apps_" + $Version` 
(see line 19854 ff)

In the ZIP archive for the version mentioned above there is no version specific topfolder any more.

This is why neverred will not find the files after extraction!

 Files and architecture folders are not located in
```diff
- `"$PSScriptRoot\$Product\$MSFSLogixChannelClear\$FolderPath"`

but are instead located directly under 

+ `"$PSScriptRoot\$Product\$MSFSLogixChannelClear"`
```

**Comment by RvT-C** [comment]
We faced the same issue. the path has change, and after an uninstall, it would not install again. 

We missed this, and we have "some" user complains.

**Comment by Deyda** [comment]
Solved with Version 2.10.22
